src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -267,7 +267,7 @@ public function add(string $body, array $headers, int $delayInMs = 0): void
                     throw new TransportException(json_last_error_msg());
                 }
 
-                $score = (int) ($this->getCurrentTimeInMilliseconds() + $delayInMs);
+                $score = $this->getCurrentTimeInMilliseconds() + $delayInMs;
```

Comment:
`getCurrentTimeInMilliseconds()` returns an int, `int $delayInMs = 0` in the args